app.py

```diff
@@ -595,6 +608,126 @@ def call_api(url, params=None):
     return data
 
 
+# get each team's rostered players
+def get_each_teams_players():
+    league_id = request.args.get('leagueId')
+    url = f'https://fantasy.espn.com/apis/v3/games/fba/seasons/{app.config.get("SEASON")}/segments/0/leagues/{league_id}'
+    params = {
+        'view': 'mRoster'
+    }
+    data = call_api(url, params)
+    team_rosters = {}
+    team_dict = get_team_dict(league_id)
+    for team in data['teams']:
+        team_roster = {}
+        team_players = team['roster']['entries']
+        for player in team_players:
+            full_name = player['playerPoolEntry']['player']['fullName']
+            team_roster[full_name] = stats_cleanup(player['playerPoolEntry']['player']['stats'])
+        # Store the roster and each player's stats in a dict of team roster by team names
+        team_rosters[team_dict[team["id"]]] = team_roster
+    return team_rosters
+
+
+def stats_cleanup(stats):
+    # The stats dict has a couple of different average stats. We can find the actual stat timeframe using the
+    # following info:
+    # seasonId: season end year
+    # stat source: 0 = actual, 1 = projected
+    # stat split type: 0 = season, 1 = 7 days, 2 = 15 days, 3 = 30 days
```

Comment:
Curious how you figured this out. I'm guessing ESPN doesn't document their APIs, right? Trial and error?